src/main/java/edu/cornell/kfs/paymentworks/businessobject/PaymentWorksFieldMapping.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.kfs.paymentworks.businessobject;
+
+import java.io.Serializable;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+public class PaymentWorksFieldMapping extends PersistableBusinessObjectBase implements Serializable {
+	
+	private static final long serialVersionUID = -2187008550220892821L;
+	private Long paymentWorksFieldMappingId;
+	private String paymentWorksFieldName;
+	private String kfsFieldName;
+	
+	public Long getPaymentWorksFieldMappingId() {
```

Comment:
An ID would never be null (we aren't letting users add records here, only edit payment works field  labels, but I will change for KFS consistency.  I probably am remember a standard we had in KC to use object Longs for ID fields.